exchanges/lbank/lbank_test.go

```diff
@@ -0,0 +1,327 @@
+	if err == nil {
+		t.Error("Test Failed - CreateOrder error cannot be nil")
+	}
+	_, err = l.CreateOrder(cp.Lower().String(), "buy", 58, 681)
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+}
+
+func TestRemoveOrder(t *testing.T) {
+	if l.APIKey == "" || l.APISecret == "" {
+		t.Skip()
+	}
+	TestSetup(t)
+	cp := currency.NewPairWithDelimiter(currency.ETH.String(), currency.BTC.String(), "_")
+	_, err := l.RemoveOrder(cp.Lower().String(), "24f7ce27-af1d-4dca-a8c1-ef1cbeec1b23")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestQueryOrder(t *testing.T) {
+	if l.APIKey == "" || l.APISecret == "" {
+		t.Skip()
+	}
+	TestSetup(t)
+	cp := currency.NewPairWithDelimiter(currency.BTC.String(), currency.USDT.String(), "_")
+	_, err := l.QueryOrder(cp.Lower().String(), "1")
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+}
+
+func TestQueryOrderHistory(t *testing.T) {
+	if l.APIKey == "" || l.APISecret == "" {
+		t.Skip()
+	}
+	TestSetup(t)
+	cp := currency.NewPairWithDelimiter(currency.BTC.String(), currency.USDT.String(), "_")
+	havealook, err := l.QueryOrderHistory(cp.Lower().String(), "1", "50")
+	if err != nil {
+		t.Error(err)
+	}
+
+	log.Println(havealook)
+}
+
+func TestGetPairInfo(t *testing.T) {
+	TestSetup(t)
+	_, err := l.GetPairInfo()
+	if err != nil {
+		t.Error("somethings wrong")
+	}
+}
+
+func TestGetOpenOrders(t *testing.T) {
+	if l.APIKey == "" || l.APISecret == "" {
+		t.Skip()
+	}
+	TestSetup(t)
+	cp := currency.NewPairWithDelimiter(currency.BTC.String(), currency.USDT.String(), "_")
+	_, err := l.GetOpenOrders(cp.Lower().String(), 1, 50)
+	if err != nil {
+		t.Error("unexpected error", err)
+	}
+}
+
+func TestUSD2RMBRate(t *testing.T) {
+	TestSetup(t)
+	_, err := l.USD2RMBRate()
+	if err != nil {
+		t.Error("wtf")
+	}
+}
+
+func TestGetWithdrawConfig(t *testing.T) {
+	TestSetup(t)
+	curr := "eth"
+	_, err := l.GetWithdrawConfig(curr)
+	if err != nil {
+		t.Error("wtf", err)
+	}
+}
+
+func TestWithdraw(t *testing.T) {
+	if l.APIKey == "" || l.APISecret == "" {
+		t.Skip()
+	}
+	TestSetup(t)
+	_, err := l.Withdraw("", "", "", "", "")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestGetWithdrawRecords(t *testing.T) {
+	if l.APIKey == "" || l.APISecret == "" {
+		t.Skip()
+	}
+	TestSetup(t)
+	_, err := l.GetWithdrawalRecords("eth", "0", "1", "20")
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestLoadPrivKey(t *testing.T) {
+	if l.APIKey == "" || l.APISecret == "" {
+		t.Skip()
+	}
+	l.SetDefaults()
+	l.APISecret = testAPISecret
+	err := l.loadPrivKey()
+	if err != nil {
+		t.Error(err)
+	}
+	l.privKeyLoaded = false
+	l.APISecret = "errortest"
+	err = l.loadPrivKey()
+	if err == nil {
+		t.Errorf("expected error due to pemblock nil, got err: %v", err)
+	}
+}
+
+func TestSign(t *testing.T) {
+	if l.APIKey == "" || l.APISecret == "" {
+		t.Skip()
+	}
+	l.SetDefaults()
+	l.APISecret = testAPISecret
+	l.loadPrivKey()
+	_, err := l.sign("wtf", l.privateKey)
```

Comment:
Change string, please make sure in future you try to capture these strings before opening PR.